### ISSUE TITLE
How to recreate the solar system

### ISSUE THREAD

**Issue by thunderbear06** [SebLague/Solar-System#26]
im aware there is a scene for the solar system that was used for the videos but i was wondering if i could make my own (if possible)
Thanks in advance!

**Comment by Delofon** [comment]
Yes, of course it is possible. The most straightforward way to do it would be to duplicate existing celestial bodies and alter their settings: the body's gameobject deals with gravity stuff and its radius, mesh holder deals with shape and shading. To edit the planet's shape and shading, I recommend creating new planet settings rather than editing existing: Right Click/Create/Celestial Body and then there's a zoo of options (start with Settings Holder, you'll finish eventually by creating and referencing other assets). You can also change scene to Planet, which makes this process a lot easier.

**Comment by ZoziCoder5150** [comment]
> Yes, of course it is possible. The most straightforward way to do it would be to duplicate existing celestial bodies and alter their settings: the body's gameobject deals with gravity stuff and its radius, mesh holder deals with shape and shading. To edit the planet's shape and shading, I recommend creating new planet settings rather than editing existing: Right Click/Create/Celestial Body and then there's a zoo of options (start with Settings Holder, you'll finish eventually by creating and referencing other assets). You can also change scene to Planet, which makes this process a lot easier.

I created a celestial body and chose settings holder now what do I do
